### PR TITLE
edited the research part of my bio

### PR DESCRIPTION
**Pull request by maria-kuruvilla**
@brendanwallace please review

### CODE REVIEW

File: content/people/mariakuruvilla.md [hunk 1]
```diff
@@ -36,4 +38,4 @@ Maria would win that award. Maria if you're reading this, want to go to Araya's
 Place with me and Ben?
 
 
-
+Here are links to Maria's [Google scholar](https://scholar.google.com/citations?user=3_d0FkoAAAAJ&hl=en) and [Github](https://github.com/maria-kuruvilla).
```
Comment (brendanwallace) [inline content/people/mariakuruvilla.md:41]: I changed this sentence

**Review by brendanwallace — APPROVED** [review]
LGTM. Please confirm the change I made looks good, then you can submit.

**Review by maria-kuruvilla — COMMENTED** [review]
Thanks!

**Comment by brendanwallace** [comment]
going to merge this for you!

**Pull request merged**